ci/tasks/build.sh

```
@@ -0,0 +1,145 @@
+    echo "environment variable $name must be set"
+    exit 1
+  fi
+}
+
+check_param IAAS
+check_param HYPERVISOR
+check_param OS_NAME
+check_param OS_VERSION
+AGENT_SUFFIX="${AGENT_SUFFIX--go_agent}"
+
+export CANDIDATE_BUILD_NUMBER=$( cat "${REPO_PARENT}/version/number" | sed 's/\.0$//;s/\.0$//' )
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)